README.md

```diff
@@ -34,6 +34,7 @@ The cookbook is a living document. :seedling:
 ### Other Built-in Functions & Operators
 1. [Working with Dates and Timestamps](other-builtin-functions/01_date_time/01_date_time.md)
 2. [Building the Union of Multiple Streams](other-builtin-functions/02_union-all/02_union-all.md)
+3. [Filter out late data](other-builtin-functions/03_current_watermark/03_current_watermark.md)
```

Comment:
```suggestion
3. [Filter out Late Data](other-builtin-functions/03_current_watermark/03_current_watermark.md)
```

Reply:
I would also say "Filtering" instead of "Filter" for consistency in tenses